includes/class-wcpdf-admin.php

```diff
@@ -600,6 +600,10 @@ public function save_invoice_number_date($post_id, $post) {
 			if ( ! isset($_POST['action']) || $_POST['action'] != 'editpost' ) {
 				return;
 			}
+			// Check if user is allowed to change invoice data
+			if ( ! apply_filters( 'wpo_wcpdf_current_user_is_allowed', current_user_can( 'manage_woocommerce' ), 'invoice' ) ) {
```

Comment:
this is more restricting than the previous behavior, which only required `edit_shop_orders` or `manage_woocommerce_orders` (the latter is a legacy capability that hasn't existed for eons).
We should not change that capability requirement as it's a breaking change (and will result in unexpected behavior for the users because they don't see any errors).

Reply:
Changed here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/356/commits/32a08bd2bbd98c02691ae1c21cd68a93467d118e

Only render icons when user can manage document here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/356/commits/f4f75a62f2b25bc9e067e2aa4813ef404bc5ff77